docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -271,238 +288,512 @@ Go ahead and review each of the following five files in the pack.
+
+The following sections will guide you through starting the registry server, logging in, pushing your custom add-on pack, and, finally, configuring the registry server in Palette.
+
+### Start the Registry Server
+
+<Tabs groupId="registry-server">
+
+<TabItem label="Spectro Registry" value="Spectro_Registry">
 
-### Start and Expose the Registry Server
-Start the registry server by issuing the following command from the bash session you opened into the tutorials container. 
 <br />
 
+Start the registry server by issuing the following command from the tutorial container bash session. 
+
 ```bash
 registry serve /etc/spectro/config.yml > /var/log/registry.log 2>&1 &
 ```
 
-The registry server will start in HTTP mode (not HTTPS). Refer to the [Add a Custom Registry](adding-a-custom-registry.md) guide to learn more about deploying an HTTPS registry server. 
+The registry server starts in HTTP mode. If you want to deploy an HTTPS registry server, refer to the [Add a Custom Registry](adding-a-custom-registry.md) guide. 
 
 
-Next, expose the registry server to the public so that you can configure it later in Palette. Use Ngrok reverse proxy to expose the registry server listening on port 5000 via an HTTP tunnel using the following command. 
-<br />
+Next, make the registry server accessible to the public using [Ngrok](https://ngrok.com/) reverse proxy so that you can configure it later in Palette. Execute the command below to expose the registry server listening on port 5000 via an HTTP tunnel. 
+
 
 ```bash
 ngrok http 5000 --log-level debug
 ```
 
-The command above will reserve the current bash session and display the status of each HTTP request made to the Ngrok server later in this tutorial. The screenshot below shows the registry server successfully exposed via Ngrok.  
+This command reserves the current bash session and displays the status of each HTTP request made to the Ngrok server. The image below shows the registry server successfully exposed via Ngrok.  
 
-<br />
 
 ![Screenshot of registry server exposed via ngrok](/tutorials/deploy-pack/registries-and-packs_deploy-pack_ngrok-start.png )
 
+
+Check if the registry server is accessible from outside the tutorial container by visiting the `/health` endpoint. Open your browser and go to *https://Your-URL-Here/health*, replacing the base URL with the Ngrok URL output. You should get a `{"status":"UP"}` response.
+
+</TabItem>
+
+<TabItem label="ECR" value="ECR_Registry">
+
+<br />
+
+An Amazon ECR private registry is provided to each AWS account, allowing the creation of repositories within it.
```

Comment:
```suggestion
```
This sentence is not quite correct, as a user would have to create their own ECR registry. Probably best to remove it.

Reply:
I used the info available on the [ECR Documentation](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html) page. We cannot create ECR registries; instead, AWS provides us with one, and we can create repositories within it. However, I agree that this sentence can confuse the user, so I removed it from the tutorial.